databook/physiology/ophys/visual-behavior/VBO-ExperimentData.md

```diff
@@ -412,12 +412,13 @@ We can see that as expected, events trace is much cleaner than dff and it genera
 
 ## Stimulus presentations
 
-The `stimulus_presentations` table contains one entry for each stimulus that was presented during the session, along with important metadata including stimulus `start_time` and `end_time`, as well as `image_name`, and whether the stimulus was an image change (`is_change` = True) or an image omission (`omitted` = True).
+The `stimulus_presentations` table contains one entry for each stimulus that was presented during the session, along with important metadata including stimulus `start_time` and `end_time`, as well as `image_name`, and whether the stimulus was an image change (`is_change` = True) or an image omission (`omitted` = True). `stimulus_block` and `stimulus_block_name` indicate the type of stimulus mice were presented at a given point in a session. To select active change detection behavior, first we need to filter the table for `stimulus_block_name` = `change_detection_behavior` or `stimulus_block` = `1`. Note that different sessions may have different number of stimulus blocks, thus `change_detection_behavior` may be assosiated with either `0` or `1` in `stimulus_block` column.
 
 Get stimulus information for this experiment and assign it to a table called `stimulus_table`
 
 ```{code-cell} ipython3
 stimulus_table = ophys_experiment.stimulus_presentations.copy()
+stimulus_table = stimulus_table[stimulus_table.stimulus_block_name=='change_detection_behavior']
 stimulus_table.head(10)
 ```
```

Comment:
so this example isn't getting stimulus information for the experiment, it's getting stimulus information for a specific block. What about the other blocks?

Reply:
also fixed (same solution as the comment above - i have it print out the different stim blocks now, with more documentation)